<launch>
	<node pkg="vicon_bridge" type="vicon_bridge" name="vicon" output="screen">
		<param name="stream_mode" value="ClientPull" type="str" />
		<param name="datastream_hostport" value="10.32.24.190:801" type="str" />
		<param name="tf_ref_frame_id" value="/world" type="str" />
		</node>
</launch>
